src/material/list/list-option.scss

```diff
@@ -54,10 +54,10 @@
 }
 
 @include cdk.high-contrast(active, off) {
-  // In single selection mode, the selected option is indicated by changing its
-  // background color, but that doesn't work in high contrast mode. We add an
-  // alternate indication by rendering out a circle.
-  .mat-mdc-list-option.mdc-list-item--selected::after {
+  // In single selection mode and for links, the active/selected option is indicated by changing its
+  // background color. Since that doesn't work in high contrast mode, we add an alternate indication
+  // by rendering a circle.
+  .mat-mdc-list-option.mdc-list-item--selected::after, a.mdc-list-item--activated::after {
```

Comment:
The CSS here is for the list-option, while the activated class is for normal list items. Maybe share this as a mixin in a separate file and use it here in `list-option.scss` and in `list.scss`?

Reply:
Okay, I moved it into a mixin in _high-contrast-list-theme.scss.

Reply:
I don't think Paul was suggesting pulling it into a theme mixin; the styles aren't really theme related. He was just saying to put it in a shared mixin that's included in list-option.scss to style list-option and list.scss to style the list-item